packages/nextjs/components/Footer.tsx

```diff
@@ -58,17 +58,17 @@ export const Footer = () => {
               </p>
               <a
                 className="flex justify-center items-center gap-1"
-                href="https://buidlguidl.com/"
+                href={projectConfig.creatorLink}
                 target="_blank"
                 rel="noreferrer"
               >
-                <BuidlGuidlLogo className="w-3 h-5 pb-1" />
-                <span className="link">BuidlGuidl</span>
+                <projectConfig.creatorSvg className="w-3 h-5 pb-1" />
+                <span className="link">{projectConfig.creator}</span>
               </a>
             </div>
             <span>·</span>
             <div className="text-center">
-              <a href="https://t.me/joinchat/KByvmRe5wkR-8F_zz6AjpA" target="_blank" rel="noreferrer" className="link">
+              <a href={projectConfig.supportLink} target="_blank" rel="noreferrer" className="link">
```

Comment:
Not sure lot's of projects has tg chats. Probably we need to remove or at least rename that link. But discuss about it first